mesonbuild/interpreterbase.py

```diff
@@ -819,7 +819,7 @@ def evaluate_indexing(self, node: mparser.IndexNode) -> TYPE_var:
     def function_call(self, node: mparser.FunctionNode) -> T.Optional[TYPE_var]:
         func_name = node.func_name
         (posargs, kwargs) = self.reduce_arguments(node.args)
-        if is_disabled(posargs, kwargs) and func_name != 'set_variable' and func_name != 'is_disabler':
+        if is_disabled(posargs, kwargs) and func_name != 'get_variable' and func_name != 'set_variable' and func_name != 'is_disabler':
```

Comment:
```suggestion
        if is_disabled(posargs, kwargs) and func_name not in {'get_variable', 'set_variable', 'is_disabler'}
```

Let's go ahead and make this a little faster and more maintainable